docs/roles/integrator/exchange-integration.md

```diff
@@ -773,9 +771,92 @@ and we can find its outcome in `receipts_outcome`:
 
 which indicates that the transaction is successful.
 
-## Block and Finality
+## Fungible tokens
+
+Please see the [spec for the fungible token standard](https://nomicon.io/Standards/FungibleToken/README.html) and an [example implementation](https://github.com/near-examples/FT) for reference details.
+
+One notable aspect of the standard is that method names are prefixed with `ft_`. This will be a helpful convention when querying for transactions related to fungible tokens.
+
+### Get balance
+
+Using the abstraction of the NEAR CLI tool, we can check the balance of a user's account with this command:
+
+`near view ft.demo.testnet ft_balance_of '{"account_id": "mike.testnet"}'`
+
+Returns:
+
+```
+View call: ft.demo.testnet.ft_balance_of({"account_id": "mike.testnet"})
+'1000000'
+```
+
+Using HTTPie:
```

Comment:
```suggestion
Alternatively, you can [call a contract function](/docs/api/rpc#call-a-contract-function) using the `query` RPC endpoint. Below is an example using Httpie:
```

Reply:
Gotta capitalize it properly to be https://httpie.io/ HTTPie 😇

Reply:
done :)